dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -125,6 +125,7 @@
 		tlb: tlb@17e000 {
 			compatible = "intel,adsp-tlb";
 			reg = <0x17e000 0x1000>;
+			label = "HPSRAM_TLB";
```

Comment:
Actually, the main branch is in the process of removing usage of `label` properties in device tree. So please change the code so this does not require this.

Reply:
@marcinszkudlinski  the project is removing the label property. Just remove it. Look the comment below about how to get the device handle without it.